src/docfx/lib/log/ErrorLevel.cs

```diff
@@ -3,8 +3,9 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal enum ReportLevel
+    internal enum ErrorLevel
     {
+        Off,
```

Comment:
>Off [](start = 8, length = 3)

What does this mean?

Reply:
Turn errors and warnings off, to support a config entry like:
```
rules:
    file-not-found: warning
    invalid-yaml-header: off
```